api/rpc/service/service.go

```diff
@@ -132,10 +138,51 @@ func (s *Service) Create(ctx context.Context, req *ServiceCreateRequest) (*Servi
 	resp := &ServiceCreateResponse{
 		Id: r.ID,
 	}
-	fmt.Printf("Service: %s created, id=%s\n", req.ServiceSpec.Name, resp.Id)
+	fmt.Printf("Service: %s created, id=%s\n", serv.Name, resp.Id)
 	return resp, nil
 }
 
+// set mount/volume info in serviceSpec
+func (s *Service)processMounts(service *swarm.ServiceSpec, mounts []string) {
+	service.TaskTemplate.ContainerSpec.Mounts = []mount.Mount{}
+	for _, vol := range mounts {
+		list := strings.Split(vol, ":")
+		ro := false
+		if len(list) == 3 && list[2] == "ro" {
+			ro = true
+		}
+		if len(list) == 1 {
+			//create a anonymous  volume
+			fmt.Println("create anonymous volume")
```

Comment:
from what I see it's still bind to a host directory, it's just implicit (source is same as the target)